### DISCUSSION TITLE
Auto-converted curly quotes in search bar

### DISCUSSION THREAD

**Discussion by UmbreonLibris** [phcode-dev/phoenix#1474] [Q&A]
This is probably a bug, but maybe there's a setting I'm missing…
The new phcode desktop app for Mac (Intel) is auto-converting some quotation marks I type into the search bar into typographer's (curly) quotes.
<img width="244" alt="image" src="https://github.com/phcode-dev/phoenix/assets/138822747/9d36b830-cd63-4e40-ab16-e47b54ef8a0d">

Can I disable this somehow? I did not have this problem with the browser app, so it seems to be specifically an issue with the desktop app.

**Comment by abose** [comment]
Thanks for raising the issue. This was fixed as part of the new find in files features https://github.com/phcode-dev/phoenix/pull/1461

The release should be available by the end of this month via automatic updates.

### About the issue root cause:
Safari has a smart quotes feature in macOS: https://support.apple.com/en-in/guide/mac-help/mh35735/mac , that was active in the search box, it will be auto disabled to fix the issue in the next release.
